setup.py

```diff
@@ -9,33 +9,20 @@ def get_requirements(filename):
         out.append(a.strip())
     return out
 
-def get_packages():
-    """Method to retrieve packages to be bundled."""
-    base_dir = '.'
-    packages = [base_dir]
-    for (path, dirs, files) in os.walk(base_dir):
-        try:
-            dirs.remove('__pycache__')
-        except ValueError:
-            pass
-        if '__init__.py' in files:
-            packages.extend([os.path.join(path, dir) for dir in dirs])
-    return packages
-
 requirements = get_requirements('requirements.txt')
 setup(
     name="pynnotate",
     version="0.2",
-    packages=get_packages(),
-    scripts=['pynnotate.py'],
+    packages=find_packages(),
```

Comment:
🤣 I wanted to try this. Found on stackoverflow. Did not understand why you imported and did not use.

Reply:
😭